documentation/project-docs/SDK-PR-guide.md

```diff
@@ -0,0 +1,56 @@
+# SDK PR Guide
+
+## Branching
+See the [versions](https://learn.microsoft.com/en-us/dotnet/core/porting/versioning-sdk-msbuild-vs#lifecycle) document for more details on how SDK versions map to MSBuild and Visual Studio.
+
```

Comment:
It may be worthwhile to mention the `alias` or places where internal folks can see the deadlines that tactics sends out. (I don't even think I get those emails still.)

Reply:
I was trying to avoid internal links in the public document. I added some text that will direct people where to look for or ask.